docs/reference/modules/node.asciidoc

```diff
@@ -15,7 +15,7 @@ All nodes know about all the other nodes in the cluster and can forward client
 requests to the appropriate node.
 
 By default, a node is all of the following types: master-eligible, data, ingest,
-and (if available) machine learning and transform.
+and (if available) machine learning.
```

Comment:
> and transform?

Is this change correct? Or are all nodes truly transform nodes by default?

Reply:
sounds good to me

Reply:
sorry, did I got this wrong? It was right before, or maybe we should say:

"By default, a node is all of the following types: master-eligible, data, ingest,
and (if available) machine learning. All data nodes are also transform nodes."

Reply:
Thanks, I've made that change now too.